search-service/src/main/kotlin/com/egm/stellio/search/service/AttributeInstanceService.kt

```diff
@@ -96,19 +97,57 @@ class AttributeInstanceService(
     ): List<AttributeInstanceResult> =
         search(temporalQuery, listOf(temporalEntityAttribute), withTemporalValues)
 
+    suspend fun selectOldestDate(
+        temporalQuery: TemporalQuery,
+        temporalEntityAttributesIds: String
+    ): ZonedDateTime {
+        var selectQuery =
+            """
+                SELECT temporal_entity_attribute,min(time) as first
```

Comment:
why are you selecting `temporal_entity_attribute` (and why the final `GROUP BY`?) ?

Reply:
I thought I had to add it to use the where. The code forced me to add the `GROUP BY` if I use the `temporal_entity_attribute` in the select